documentation/docs/configuration/plugins/git-revision.md

```diff
@@ -122,12 +122,26 @@ theme:
     - revision.history  
 ```
 
+## Codeberg
+### MkDocs projet in root directory
+```yaml
+repo_url: https://codeberg.org/exampleUser/exampleRepository
+edit_url_template: _edit/main/docs/{path}
```

Comment:
nice to include the `edit_url_template` attribute even though`mkdocs-terminal` theme doesn't support it yet
thanks!